HealthcareSystem/Backend/Controller/NotificationSurveyAndFeedback/FeedbackController.cs

```diff
@@ -14,22 +15,22 @@ namespace Controller.NotificationSurveyAndFeedback
    public class FeedbackController
    {
 
-        private FeedbackService feedbackService = new FeedbackService();
-        public void NewFeedback(Feedback feedback)
+        private FeedbackService _feedbackService = new FeedbackService(new MySqlFeedbackRepository());
```

Comment:
I would keep the service connected to the file repository for now instead, since there is no database yet, and you'll need to access a repository that's actually implemented.